nipype/interfaces/base.py

```diff
@@ -1063,13 +1055,19 @@ def run(self, **inputs):
         results :  an InterfaceResult object containing a copy of the instance
         that was executed, provenance information and, if successful, results
         """
+        from ..utils.profiler import resource_monitor, ResourceMonitor
+
+        enable_rm = resource_monitor and getattr(self.inputs, 'resource_monitor', True)
```

Comment:
should this be `getattr(self, 'resource_monitor', True)`?

assuming it should be a class attribute rather than an input.